assets/trix/images/heading.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generated by IcoMoon.io -->
```

Comment:
Good to stub this toolbar icon out for now, but let’s make sure to have @early design a real one for us.